src/styles/blocks/stars--0.css

```diff
@@ -0,0 +1,44 @@
+.stars--0 {
+  display: flex;
+  margin-top: 22px;
+}
+
+.stars--0 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
It seems like you have repeated code for each `.stars__star` element. Consider using a single class to apply these common styles to avoid repetition and make your CSS more maintainable.